loader.php

```diff
@@ -144,8 +142,13 @@ function enqueue_assets( $directory, $opts = [] ) {
 	}
 
 	if ( empty( $assets ) ) {
-		// TODO: This should be an error condition.
-		return;
+		if ( WP_DEBUG ) {
+			handle_assets_error();
+		}
+		else {
+			// Trigger a warning, but otherwise do nothing.
+			trigger_error( 'React WP Scripts Error: Unable to find React asset manifest', E_USER_WARNING );
+		}
```

Comment:
Should this also `return`? We could probably always `trigger_error` to avoid the `else` as well.

Reply:
Solved!